apps/main/src/lend/store/createAppSlice.ts

```diff
@@ -36,18 +37,10 @@ const createAppSlice = (set: StoreApi<State>['setState'], get: StoreApi<State>['
 
     // reset store
     if (isNetworkSwitched) {
-      Object.keys(get()).forEach((stateKey) => {
-        if (
-          stateKey.startsWith('loan') ||
-          stateKey.startsWith('user') ||
-          stateKey === 'tokens' ||
-          stateKey === 'chartBands' ||
-          stateKey === 'campaigns'
-        ) {
-          // @ts-ignore
-          if ('resetState' in get()[stateKey]) get()[stateKey].resetState()
-        }
-      })
+      recordEntries(get())
+        .filter(([stateKey]) => stateKey.startsWith('loan') || stateKey.startsWith('user') || stateKey === 'chartBands')
```

Comment:
Campaign slice is gone but I believe we still have a token slice, which now would be reset. Might be ok, dunno